docker-images/build-images.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+KAFKA_VERSIONS=$(cat docker-images/kafka.version)
+KAFKA_MODULES=$(ls -d kafka/*/)
+HTTP_MODULES=$(ls -d http/*/)
+MODE=${1:-"all"}
```

Comment:
what's the difference between `build` and `all`?

Reply:
`build` will just execute mvn build and docker build to check that everything is ready to go before push
`all` will do the `build`, `push` and `clean`

Reply:
I had there just `make all` firstly, but then I imagine that you would like to try to build it with all versions (specified in `kafka.version`) in case there is some change that will break the build.